recipe/meta.yaml

```
@@ -13,27 +13,15 @@ source:
 
 build:
   number: 0
-  features:
-    - vc9   # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py>=35]
+
 
 requirements:
   build:
-    - python           # [win]
-    - vc 9             # [win and py27]
-    - vc 10            # [win and py34]
-    - vc 14            # [win and py>=35]
-    - posix            # [win]
-    - pthreads-win32   # [win]
-    - m2w64-toolchain  # [win]
-    - toolchain
-    - zlib 1.2.*
+    - {{ compiler('c') }}
```

Probably need this for Windows.

```suggestion
    - {{ compiler('c') }}               # [unix]
    - {{ compiler('m2w64_c') }}         # [win]
```